syntax="proto3";

// buf:lint:ignore PACKAGE_VERSION_SUFFIX
package airliner_grpc;


// buf:lint:ignore SERVICE_SUFFIX
service UserValidationForTokenGeneration{
  // buf:lint:ignore RPC_REQUEST_STANDARD_NAME
  // buf:lint:ignore RPC_RESPONSE_STANDARD_NAME
  rpc ValidateUserCredentials(TokenReqMessage) returns (TokenResMessage){}
};

message TokenReqMessage{
  string userid = 1;
  string username = 2;
  string passcode = 3;
}

message TokenResMessage{
  string userid = 1;
  bool isvalid = 2;
}




